java/core/src/main/java/co/worklytics/psoxy/rules/msft/PrebuiltSanitizerRules.java

```diff
@@ -275,6 +292,219 @@ public class PrebuiltSanitizerRules {
+        .build();
+
+    static final Endpoint MS_TEAMS_COMMUNICATIONS_CALLS = Endpoint.builder()
+        .pathTemplate(MS_TEAMS_PATH_TEMPLATES_COMMUNICATIONS_CALLS)
+        .transform(Transform.Redact.builder()
+            .jsonPath("$..displayName")
+            .build())
+        .allowedQueryParams(List.of("$select","$top","$expand"))
+        .build();
+
+    static final Endpoint MS_TEAMS_COMMUNICATIONS_CALL_RECORDS = Endpoint.builder()
+        .pathRegex(MS_TEAMS_PATH_TEMPLATES_COMMUNICATIONS_CALL_RECORDS_REGEX)
+        .transform(Transform.Redact.builder()
+            .jsonPath("$..user.displayName")
+            .jsonPath("$..reflexiveIPAddress")
+            .jsonPath("$..relayIPAddress")
+            .jsonPath("$..ipAddress")
+            .jsonPath("$..subnet")
+            .jsonPath("$..macAddress")
+            .jsonPath("$..caller.name")
+            .jsonPath("$..callee.name")
+            .jsonPath("$..captureDeviceName")
+            .jsonPath("$..renderDeviceName")
+            .build())
+        .allowedQueryParams(List.of("$select","$expand"))
+        .build();
+
+    static final Endpoint MS_TEAMS_COMMUNICATIONS_CALL_RECORDS_GET_DIRECT_ROUTING_CALLS = Endpoint.builder()
+        .pathTemplate(MS_TEAMS_PATH_TEMPLATES_COMMUNICATIONS_CALL_RECORDS_GET_DIRECT_ROUTING_CALLS)
+        .transform(Transform.Redact.builder()
+            .jsonPath("$..value[*].userPrincipalName")
+            .jsonPath("$..value[*].userDisplayName")
+            .jsonPath("$..value[*].callerNumber")
+            .jsonPath("$..value[*].calleeNumber")
+            .build())
+        .allowedQueryParams(List.of("$skip"))
+        .build();
+
+    static final Endpoint MS_TEAMS_COMMUNICATIONS_CALL_RECORDS_GET_PSTN_CALLS = Endpoint.builder()
+        .pathTemplate(MS_TEAMS_PATH_TEMPLATES_COMMUNICATIONS_CALL_RECORDS_GET_PSTN_CALLS)
+        .transform(Transform.Redact.builder()
+            .jsonPath("$..value[*].userPrincipalName")
+            .jsonPath("$..value[*].userDisplayName")
+            .jsonPath("$..value[*].callerNumber")
+            .jsonPath("$..value[*].calleeNumber")
+            .build())
+        .allowedQueryParams(List.of("$skip"))
+        .build();
+
+    static final Endpoint MS_TEAMS_USERS =  Endpoint.builder()
+        .pathTemplate(MS_TEAMS_PATH_TEMPLATES_USERS)
+        .transform(Transform.Redact.builder()
+            .jsonPath("$..businessPhones")
+            .jsonPath("$..displayName")
+            .jsonPath("$..givenName")
+            .jsonPath("$..mail")
+            .jsonPath("$..mobilePhone")
+            .jsonPath("$..surname")
+            .jsonPath("$..userPrincipalName")
+            .jsonPath("$..signInActivity")
+            .jsonPath("$..identities")
+            .build())
+        .allowedQueryParams(List.of("$select","$top","$filter", "$orderBy", "$count", "$expand", "$search"))
```

Comment:
```suggestion
        .allowedQueryParams(List.of("$select","$top","$filter", "$orderby", "$count", "$expand", "$search"))
```

Watch out here; parameters should match the original specification case, otherwise rules will not work (this on the changes I have to do on #593)